README.md

```diff
@@ -2,6 +2,43 @@
 
 Automate onboarding with https://github.com/packit/dist-git-to-source-git
 
+## Decommissioning
+
+**As of Q1 2022, we are planning to decommission the `c8` branches and repositories.**
+
+This should happen during 3 stages.
+
+### Stage 1
+
+1. Identify c8-only repos (see [packages.json](packages.json))
```

Comment:
Step 1.1: Send an email including the list of repositories to be archived and deleted (or a link to them, the putting it into the email probably would be better as engineers might have filters set up to catch communication involving their packages) to the CentOS and RHEL devel. The list of repositories where the `c8[s]` branch is planned to be removed should also be communicated.

Reply:
:+1: